src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -727,6 +728,7 @@ public synchronized StringBuffer repeat(int codePoint, int count) {
     @Override
     public synchronized StringBuffer repeat(CharSequence cs, int count) {
         super.repeat(cs, count);
+        toStringCache = null;
```

Comment:
Ditto, clear toStringCache before the modification.

Reply:
Changed